css/properties.json

```diff
@@ -5833,6 +5833,21 @@
     "order": "lengthOrPercentageBeforeKeywords",
     "status": "standard"
   },
+  "text-justify": {
+    "syntax": "auto | distribute | inter-character | inter-word | none",
```

Comment:
The [draft spec](https://drafts.csswg.org/css-text-3/#text-justify-property) defines as ``auto | none | inter-word | inter-character``. Is ``distribute`` a valid value? Why change the order?